### DISCUSSION TITLE
Deleting users

### DISCUSSION THREAD

**Discussion by mithat** [danpros/htmly#810] [Q&A]
I've seen a reference to an ability to delete users being added to the admin panel, but it either doesn't exist or I'm too stupid to find it!

If it's the former, can I delete users simply by removing their file from `htmly/config/users` and their `author.md` file under `content/<username>`?

If it's the latter, what am I missing?

**Comment by Joduai** [comment]
While logged to admin panel you can delete user accounts with author and editor roles. Deleting admin type accounts is not possible under admin panel.   
But, as you mentioned, you can freely delete related user file located in config/users dir.
